lib/ht_sip_validator/validator/ocr.rb

```diff
@@ -2,6 +2,26 @@
 
 # namespace for validators for meta.yml
 module HathiTrust::Validator::OCR
+  # Returns an error for each file in the set 'other' that isn't in the set 'base'
+  # using the given names of the sets and file extension for the 'other' set
+  def file_set_diff(base, base_name, other, other_name, other_ext, message_level)
```

Comment:
Getting passed six require parameters seems like this is doing too many things, perhaps it could split into two functions.  The first `file_set_diff` that returns an array of differences, and the second generates the specific error for a given file.
```ruby
base_sequence = sequence_map(:image)
sequence_diffs = file_set_diff(base, sequence_map(:ocr))
errors = sequence_diffs.each{|seq| make_error(base[seq])}
```
```ruby
warnings = sequence_diffs.each{|seq| make_warning(base[seq])}
```

Also, that would allow each Validator to have the method that creates the appropriately formatted error associated with that validator.  If you wanted to avoid having to repeat the same human message or extras hash, could add a convenience method for `format_human_message` in order to get uniformity there.  

This could result in less dry code, but would uncouple getting the differences and generating the error messages.